### DISCUSSION TITLE
RSS of all new songs in Mirlo

### DISCUSSION THREAD

**Discussion by simonv3** [funmusicplace/mirlo#929] [Ideas]
If an artist gives permission, this RSS feed could contain a bit of music to play, but the main thing is to show new releases. 

* [ ] New songs
* [x] New releases
* [ ] New blog posts

**Comment by thevril** [comment]
I know that actually there is the possibility to subscribe via RSS to posts, but I think it’s more important to have the possibility to subscribe via RSS feed to artists or normal users, to stay updated on new uploads and user music suggestions.

**Comment by ethicalrevolution** [comment]
An RSS feed for new releases would be a really useful addition for us at https://nham.co.uk - we curate links to new releases from Fedi artists on fair music platforms. Other fair platforms such as Bandwagon have an rss feed meaning we get notified whenever there's a new release, but presently for Mirlo we have to remember to manually visit the new releases page to check if there have been any new releases. This means we're often reporting on new releases when they're less new.

**Reply by simonv3** [reply]
Releases at least will be in an RSS feed once this gets merged https://github.com/funmusicplace/mirlo/pull/1076/commits/ab80361191efec7fe4e8309d0d8e45905061363b
